liberty-remote/JakartaEE9_README.md

```diff
@@ -1,12 +1,12 @@
-# Arquillian Liberty Remote with Jakarta EE 9 and 10
+# Arquillian Liberty Remote with Jakarta EE 9, 10 and 11
```

Comment:
same comment about informing the user about Java 8 support being dropped

Reply:
Something added